docs/workflows/bind-passkey.md

```diff
@@ -18,6 +18,8 @@ In order to authenticate with Beyond Identity, you need a [Universal Passkey](..
 
 Passkeys are generated through a binding job. On creation of a binding job, a binding link will be generated. That binding link can be used to bind a passkey to a specific device or browser. This passkey will be [stored](../platform-overview/passkeys-and-devices/how-are-keys-stored) in the user's device's hardware root of trust (i.e. secure enclave).
 
+Make sure you have set up an application and configured your [Authenticator Config](../platform-overview/authenticator-config.md) before following this guide.
```

Comment:
I think this is fine for now. But we'll want to link to "configuring an application" when that workflow is ready

Reply:
Ok, I added a note to the JIRA ticket https://beyondidentity.atlassian.net/browse/DEVX-1172